src/features/nft/nftModals/reassignTagModal.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+// Components
+import { Modal } from '../../../common/components/Modal'
+import MultiSelect, {
+  OptionType,
+} from '../../../common/components/MultiSelect/MultiSelect'
+import { CircleAddButton, Button } from '../../../common/components/Buttons'
+import Dropdown from '../../../common/components/Dropdown'
+import CreateTag from '../../../common/components/CreateTag'
+
+export type TReassignTagModal = {
+  isOpen: boolean
+  handleClose?: (event: React.MouseEvent<HTMLButtonElement>) => void
+  title?: string
+}
+
+const ReassignTagModal = ({
+  title,
+  isOpen,
+  handleClose,
+}: TReassignTagModal): JSX.Element => {
+  const [isCreateOpen, setIsCreateOpen] = React.useState<boolean>(false)
```

Comment:
Can we use `useState` instead of `React.useState`? Please also update for below sections